lagom-cloud-object-storage-example/account-api/src/main/java/com/lightbend/lagom/account/api/AccountService.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.account.api;
+
+import static com.lightbend.lagom.javadsl.api.Service.named;
+import static com.lightbend.lagom.javadsl.api.Service.pathCall;
+
+import akka.NotUsed;
+import com.lightbend.lagom.javadsl.api.Descriptor;
+import com.lightbend.lagom.javadsl.api.Service;
+import com.lightbend.lagom.javadsl.api.ServiceCall;
+
+public interface AccountService extends Service {
+
+  ServiceCall<Transaction, NotUsed> deposit(String accountNumber);
+
+  ServiceCall<Transaction, NotUsed> withdraw(String accountNumber);
+
+  ServiceCall<NotUsed, Double> balance(String accountNumber);
+
+  ServiceCall<NotUsed, String> extract(String accountNumber, int extractNumber);
```

Comment:
True. I didn't though about the content type. The reason why left it as a String is because originally I was retrieving a plain text from COS. Something like:
```
ACCOUNT: abc
EXTRACT NUMBER: 1
BALANCE 1000
...
```

This involved later to json. The type here must be the `api.Extract`